x-pack/plugins/ml/common/constants/key_codes.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+
+export const KEY_CODES = {
```

Comment:
Could you use `eui/src/services/key_codes.js` from eui instead? It has all the key codes you need I think.